### PR TITLE
ITN Chainspec

### PR DESCRIPTION
**Pull request by maxsam4**
There are a few internal changes:
- Rename mainnet to itn
- Reorder itn pallets
- Fixed systematic cdd claims
- Updated weights
- Fix network detection for buffron
- Fix bug in did registration during genesis

External changes:

### new features

- ITN Genesis Config
- Primary key of every new Identity gets seeded with some POLYX

### CODE REVIEW

File: src/chain_spec.rs [hunk 1]
```diff
@@ -210,6 +213,24 @@ fn polymath_mem(n: u8) -> GenesisIdentityRecord<AccountId> {
...
+    let stripped_hash = &hash[offset..];
+    let hash_vec: Vec<u8> = rustc_hex::FromHex::from_hex(stripped_hash)
+        .expect("Failed to decode transaction hash (Invalid hex Value)");
+    let hash_array: [u8; 32] = hash_vec
+        .try_into()
+        .expect("Failed to decode transaction hash (Invalid hash length)");
+    (nonce, hash_array.into())
+}
+
+fn generate_bridge_locks(count: u32) -> Vec<(u32, H256)> {
```
Comment (vkomenda) [inline src/chain_spec.rs:230]: It makes sense to define a struct with named fields and some methods for the element type `(u32, H256)`.

File: pallets/bridge/src/lib.rs [hunk 2]
```diff
@@ -330,10 +331,10 @@ decl_storage! {
                     tx_hash: tx.tx_hash,
                 };
                 debug::info!(
-                    "Genesis bridge transaction to {:?} with nonce {} for {:?} POLYX",
+                    "Credited Genesis bridge transaction to {:?} with nonce {} for {:?} POLYX",
```
[the remark this answers is not in the document]
Comment (maxsam4) [reply pallets/bridge/src/lib.rs:334]: We are actually credited the POLYX amount to the user here. The transaction was created earlier.

**Review by vkomenda — APPROVED** [review]

File: pallets/weights/src/pallet_staking.rs [hunk 3]
```diff
@@ -8,210 +8,210 @@ use polymesh_runtime_common::{RocksDbWeight as DbWeight, Weight};
...
+        (8_079_708_000 as Weight)
+            .saturating_add((50_218_000 as Weight).saturating_mul(s as Weight))
             .saturating_add(DbWeight::get().reads(1 as Weight))
             .saturating_add(DbWeight::get().writes(1 as Weight))
     }
-    fn payout_stakers(n: u32) -> Weight {
-        (267_975_000 as Weight)
-            .saturating_add((123_000 as Weight).saturating_mul(n as Weight))
+    // WARNING! Some components were not used: ["n"]
+    fn payout_stakers(_: u32) -> Weight {
```
Comment (fmiguelgarcia) [inline pallets/weights/src/pallet_staking.rs:117]: Not sure if parameter `n` makes sense here.
If `MaxNominatorRewardedPerValidator` is updated by a Runtime update, we could also update the this file according to that.

**Review by fmiguelgarcia — APPROVED** [review]
ILGTM, some comments to be handled in separated tasks.

**Pull request merged**